coords_systems.md

```diff
@@ -0,0 +1,85 @@
+
+## Image Coordinate System
+Once rasterisation is complete, the final multi-channel image will be in the image space. This is a 2D space where (0,0)
+is located in the top-left corner.
+
+If you're using one of our high-level dataset objects (either `EgoDataset` or `AgentDataset`) to generate samples, you can 
+access the world-to-image matrix using the `world_to_image` key on the returned dict. When building this matrix, several steps are combined:
+- Translate world to ego by applying the negative `ego_translation`;
+- Rotate counter-clockwise by negative `ego_yaw` to align world such that ego faces right in the image;
+- Scale from meters to pixels based on the value `pixel_size` set in the configuration;
+- Translate again such that the ego is aligned to the value `ego_center` in the configuration.
+ 
+ Note: we ignore the z coordinate for these transformation
+
+With this matrix, you can transform a point from world to image space and vice versa using its inverse.
+
+One application of this is drawing trajectories on the image:
+
+```python
+data = dataset[0]  # this comes from either EgoDataset or AgentDataset
+
+im = dataset.rasterizer.to_rgb(data["image"].transpose(1, 2, 0))  # convert raster into rgb
+
+# transform from world meters into image pixels
+positions_pixels = transform_points(data["target_positions"] + data["centroid"][:2], data["world_to_image"])
+draw_trajectory(im, positions_pixels, data["target_yaws"], TARGET_POINTS_COLOR)
+```
+
+## Satellite Coordinate System
+Satellite information is stored as an RGB image in the `aerial_map` folder of the dataset. Together with that we provide
+a matrix to convert from the [ECEF](https://en.wikipedia.org/wiki/ECEF) reference system to this image reference system (i.e. how to convert a XYZ ECEF coord into a 2D pixel coord).
+
+Because the `.zarr` stores information in the world reference system, an additional conversion is usually required:
+- world coordinates must be converted into ECEF coordinates. This transformation matrix is currently hard-coded but will be shipped with the dataset
+in the future. It is **dataset dependent** as it encodes where the dataset world origin is located in the Earth frame;
+- ECEF coordinates must be converted into the aerial image reference system using the above mentioned matrix.
+
+The `SatelliteRasterizer` and its derived classes combine these two matrix into a single one and directly convert
+world coordinates from the `.zarr` into 2D pixels coordinates.
+
+## Semantic Coordinate System
+Semantic information is stored as a protobuf file. The protobuf store information as a list of elements of different types (e.g lanes, crosswalks, etc).
+
+Each elements can have one or multiple geometric features (e.g. the left and right delimiting lines for a lane) which are described
+as a list of 3D points.
+
+Each elements features are localised in its own reference system, which is itself geo-localised:
```

Comment:
element

Reply:
Each elements features are localised in its own `local` reference system